src/wireviz/Harness.py

```diff
@@ -20,11 +21,12 @@
     open_file_read, open_file_write
 
 
+@dataclass
 class Harness:
+    metadata: Metadata
```

Comment:
```suggestion
    metadata: Dict
```

Reply:
I suggest defining `Metadata = Dict` in `DataClasses.py` to make it easy to expand it to a more advanced type later if needed, and to enable continue using `Metadata` as argument type hints, e.g. in `generate_html_output()`.

Reply:
OK! That sounds like a good compromise.